textdb/textdb-sandbox/team3datafile.txt

```diff
@@ -0,0 +1,4 @@
+networkx 1.1.1
```

Comment:
@chenlica .. I am talking about creating source folder under textdb-sandbox project. Every project will have its own src/main/resources and src/test/resources source folder and the corresponding data files of that project will be included in these folders.
I created pull request #78 after modifying @laisycs code to use the folder structure I suggested. Please review that once.